src/frontend/js/modules/i18n/i18n.filter.spec.js

```diff
@@ -0,0 +1,19 @@
+/* global chai: false */
+
+const { expect } = chai;
+const { module, inject } = angular.mock;
+
+describe('the esnI18n filter', function() {
+
+  beforeEach(module('esn.i18n'));
+
+  it('should sanitize string',
+    inject(function(esnI18nFilter) {
```

Comment:
Nit: This should be injected on `beforeEach` block and we don't need to inject it in each test case.